packages/ui-components/src/Input/EmailPasswordInput.tsx

```diff
@@ -0,0 +1,31 @@
+import React from 'react';
+import { BaseTextInput, AllProps } from "./TextTypeInput";
+import "./EmailPassword.scss";
+
+export type ExistingPasswordProps = Omit<AllProps, "multiline">;
+export type NewPasswordProps = Omit<ExistingPasswordProps, "forgotPasswordLinkDiv">;
+export type EmailNumberProps = NewPasswordProps;
+
+export const EmailInput: React.FC<EmailNumberProps> = ({
+  ...props
+}) => {
+  return (
+    <BaseTextInput {...props}/>
+  );
+};
+
+export const NewPasswordInput: React.FC<NewPasswordProps> = ({
+  ...props
+}) => {
+  return (
+    <BaseTextInput {...props}/>
+  );
+};
+
+export const ExistingPasswordInput: React.FC<ExistingPasswordProps> = ({
```

Comment:
I find it confusing to have two separate components for a _NewPassword_ and an _ExistingPassword_. If the only real difference is the _forgot password_ link, then I think that could be a prop. Going a step further, why do we even need a boolean? If the user supplies a _forgot password_ element, then it should be displayed otherwise not. Then we could simply have a `<PasswordInput>` component.

Reply:
I remember discussing this in our meeting yesterday, but just to ensure that it's noted here for documentation:
The differences between Existing/New Password are:
1. New password can render as a text/password type, depending on if the icon is set to show/hide the password
2. suffix icon is provided in new password, to toggle the type
3. forgot password link is in existing password component
4. new password has text underneath the field to specify whether the password passes validation or not (EmailPasswordInput.tsx has the text and the styling associated with this).

Reply:
These differences weren't in the original few commits, but were realized after I tried to integrate the components into console.